src/ripple/protocol/Feature.h

```diff
@@ -34,6 +34,52 @@ uint256
 feature (const char* name);
 /** @} */
 
+constexpr size_t FeatureBitsetSize = 32;
```

Comment:
If the `assert` is checked unconditionally on startup, I vote to stick with the `assert`, because CI will help prevent a failure in this from ever making it into the wild.